builders/get-started/eth-compare/tx-fees.md

```diff
@@ -0,0 +1,192 @@
+
+=== "EIP1559"
+    ```
+    Transaction Fee =（Base Fee + Max Priority Fee Per Gas) * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+=== "Legacy"
+    ```
+    Transaction Fee = Gas Price * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+=== "EIP2930"
+    ```
+    Transaction Fee = Gas Price * Transaction Weight / {{ networks.moonbase.tx_weight_to_gas_ratio }}
+    ```
+
+The values of `Gas Price` and `Max Priority Fee Per Gas` for the applicable transaction types can be read from the block JSON object according to data structure described in [the Sidecar API page](/builders/build/substrate-api/sidecar/#evm-fields-mapping-in-block-json-object){target=_blank}. 
+
+The `Base Fee`, introduced in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559){target=_blank}, is a value set by the network itself. The `Base Fee` for `EIP1559` type transactions is currently static on Moonbeam networks and has the following assigned value:
+
+=== "Moonbeam"
+    | Variable |  Value   |
+    |:--------:|:--------:|
+    | Base fee | 100 Gwei |
+
+=== "Moonriver"
+    | Variable | Value  |
+    |:--------:|:------:|
+    | Base fee | 1 Gwei |
+
+=== "Moonbase Alpha"
+    | Variable | Value  |
+    |:--------:|:------:|
+    | Base fee | 1 Gwei |
+
+`Transaction Weight` is a Substrate mechanism used to manage the time it takes to validate a block. For all transactions types, `Transaction Weight` can be retrieved under the event of the relevant extrinsic where the `method` field is set to: 
+
+```
+pallet: "system", method: "ExtrinsicSuccess" 
+```
+
+And then `Transaction Weight` is mapped to the following field of the block JSON object:
+
+```
+extrinsics.{extrinsic number}.events.{event number}.data.0.weight
+```
+
+### Key Differences with Ethereum {: #ethereum-api-transaction-fees} 
+
+As seen in the above section, there are some key differences between the transaction fee model on Moonbeam and the one on Ethereum that developers should be mindful of when developing on Moonbeam:
+
+  - The network base fee on Moonbeam networks is currently static. This has many implications, one of which is that transaction sent with gas price set to a value lower than the network base fee will always fail, even if the blocks aren't full currently on the network. This behavior is different from Ethereum, which does not have a floor on the gas price of a transaction to be accepted. 
+    
+    The network base fee could be changed to be variable in a future runtime update. 
+
+  - The amount of gas used in Moonbeam's transaction fee model is mapped from the transaction's Substrate extrinsic weight value via a fixed factor of {{ networks.moonbase.tx_weight_to_gas_ratio }}. This value is then multiplied with the unit gas price to calculate the transaction fee. This fee model means it can be potentially significantly cheaper to send transactions such as basic balance transfers via the Ethereum API than the Substrate API. 
```

Comment:
```suggestion
  - The amount of gas used in Moonbeam's transaction fee model is mapped from the transaction's Substrate extrinsic weight value via a fixed factor of {{ networks.moonbase.tx_weight_to_gas_ratio }}. This value is then multiplied with the unit gas price to calculate the transaction fee. This fee model means it can potentially be significantly cheaper to send transactions such as basic balance transfers via the Ethereum API than the Substrate API. 
```